packages/angular-material/test/object-control.spec.ts

```diff
@@ -113,7 +117,11 @@ describe('Object Control', () => {
     })
       .overrideModule(BrowserDynamicTestingModule, {
         set: {
-          entryComponents: [TextControlRenderer, VerticalLayoutRenderer]
+          entryComponents: [
+            TextControlRenderer,
+            VerticalLayoutRenderer,
+            GroupLayoutRenderer
```

Comment:
Can we have an assert condition that is explicit about the change? It seems rather funny to me that adding the GroupLayoutRenderer fixes the issue. Maybe we could check whether the renderered result contains a Group/Card component?